internal/http_server.go

```diff
@@ -90,11 +90,21 @@ func RunHttpServer(
 		}
 		defer logFile.Close()
 
+		// XXX This is to simulate log coming from nginx to test regex banner
```

Comment:
Please be aware that this section of code is used only when `config.StandaloneTesting` = true. I changed the format here so it matches the one with have in nginx